examples/horovod/tf_hvd_simple.py

```diff
@@ -0,0 +1,125 @@
+
+import horovod.tensorflow as hvd  # noqa: E402
+import tensorflow as tf  # noqa: E402
+
+import nvtabular as nvt  # noqa: E402
+from nvtabular.framework_utils.tensorflow import layers  # noqa: E402
+from nvtabular.loader.tensorflow import KerasSequenceLoader  # noqa: E402
+
+parser = argparse.ArgumentParser(description="Process some integers.")
+parser.add_argument("--dir_in", default=None, help="Input directory")
+parser.add_argument("--b_size", default=None, help="batch size")
+parser.add_argument("--cats", default=None, help="categorical columns")
+parser.add_argument("--cats_mh", default=None, help="categorical multihot columns")
+parser.add_argument("--conts", default=None, help="continuous columns")
+parser.add_argument("--labels", default=None, help="continuous columns")
+args = parser.parse_args()
+
+
+BASE_DIR = args.dir_in or "/raid/criteo/tests/jp_movie/"
+BATCH_SIZE = args.b_size or 16384  # Batch Size
+CATEGORICAL_COLUMNS = args.cats or ["movieId", "userId"]  # Single-hot
+CATEGORICAL_MH_COLUMNS = args.cats_mh or ["genres"]  # Multi-hot
+NUMERIC_COLUMNS = args.conts or []
+TRAIN_PATHS = sorted(glob.glob(BASE_DIR + "*.parquet"))  # Output from ETL-with-NVTabular
+print(TRAIN_PATHS, BASE_DIR)
+hvd.init()
+
+print("U GOT WHAT I NEED: " + str(hvd.local_rank()))
+
+proc = nvt.Workflow.load(BASE_DIR + "workflow/")
+EMBEDDING_TABLE_SHAPES = nvt.ops.get_embedding_sizes(proc)
+
+# if gpus:
+#    tf.config.experimental.set_visible_devices(gpus[hvd.local_rank()], 'GPU')
+# dev_dev = hvd.local_rank() if hvd.local_rank() == 1 else 2
+# cupy.cuda.Device(dev_dev).use()
+train_dataset_tf = KerasSequenceLoader(
+    TRAIN_PATHS,  # you could also use a glob pattern
+    batch_size=BATCH_SIZE,
+    label_names=["rating"],
+    cat_names=CATEGORICAL_COLUMNS + CATEGORICAL_MH_COLUMNS,
+    cont_names=NUMERIC_COLUMNS,
+    engine="parquet",
+    shuffle=False,
+    buffer_size=0.06,  # how many batches to load at once
+    parts_per_chunk=1,
+    global_size=hvd.size(),
+    global_rank=hvd.rank(),
+)
+inputs = {}  # tf.keras.Input placeholders for each feature to be used
+emb_layers = []  # output of all embedding layers, which will be concatenated
+for col in CATEGORICAL_COLUMNS:
+    inputs[col] = tf.keras.Input(name=col, dtype=tf.int32, shape=(1,))
+# Note that we need two input tensors for multi-hot categorical features
+for col in CATEGORICAL_MH_COLUMNS:
+    inputs[col + "__values"] = tf.keras.Input(name=f"{col}__values", dtype=tf.int64, shape=(1,))
+    inputs[col + "__nnzs"] = tf.keras.Input(name=f"{col}__nnzs", dtype=tf.int64, shape=(1,))
+for col in CATEGORICAL_COLUMNS + CATEGORICAL_MH_COLUMNS:
+    emb_layers.append(
+        tf.feature_column.embedding_column(
+            tf.feature_column.categorical_column_with_identity(
+                col, EMBEDDING_TABLE_SHAPES[col][0]  # Input dimension (vocab size)
+            ),
+            # EMBEDDING_TABLE_SHAPES[col][1]                     # Embedding output dimension
+            16,
```

Comment:
Should probably roll back to using the original `EMBEDDING_TABLE_SHAPES` instead of the reduced dimension I stuck in there.